pkgs/by-name/rc/rcp/package.nix

```diff
@@ -30,5 +36,7 @@ rustPlatform.buildRustPackage rec {
     license = with licenses; [ mit ];
     mainProgram = "rcp";
     maintainers = with maintainers; [ wykurz ];
+    # = note: Undefined symbols for architecture x86_64: "_utimensat"
+    broken = stdenv.isDarwin && stdenv.isx86_64;
```

Comment:
Perhaps using `darwin.apple_sdk_11_0.{frameworks,rustPlatform}` might help?

Reply:
@wegank , unfortunately that does not seem to solve it.

There are two packages marked as broken due to `_utimensat`:
- rustpython
- rubyfmt

Interestingly enough, a patch in `qemu` from some time ago says it should have been fixed in SDK 11:

```nix
# QEMU upstream does not demand compatibility to pre-10.13, so 9p-darwin
# support on nix requires utimensat fallback. The patch adding this fallback
# set was removed during the process of upstreaming this functionality, and
# will still be needed in nix until the macOS SDK reaches 10.13+.
```

Reply:
Oh, `rustpython` is broken again? Then `darwin.apple_sdk_11_0.rustPlatform` is probably fundamentally broken, I suppose...

Reply:
Indeed, it was marked as broken on darwin-x86_64 about 6 months ago: https://github.com/NixOS/nixpkgs/commit/b27c3af657c9cc41751fd756d1480c783532fb11

I will give `darwin.apple_sdk_11_0.rustPlatform` a try in #312508, maybe it'll work there.